src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1144,6 +1136,19 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			item.renderType = 0;
 			item.customModelName = "Normal";
 			return item;
+		} else if (ModElementType.ITEMEXTENSION.equals(modElement.getType())){
+			ItemExtension itemExtension = new ItemExtension(modElement);
+			itemExtension.item = new MItemBlock(modElement.getWorkspace(), getRandomMCItem(random, blocksAndItems).getName());
+
+			itemExtension.enableFuel = !emptyLists;
+			itemExtension.fuelPower = new int[] { 0, 100, 1000, 4000 }[valueIndex];
+			itemExtension.isCompostable = emptyLists && _true;

```

Comment:
```suggestion
```

Reply:
I totally forgot to check tests when i removed it 😅